source/codegen/templates/service.h.mako

```diff
@@ -95,6 +96,9 @@ private:
 % for resource_handle_type in resource_repository_deps:
   ${resource_repository_deps[resource_handle_type].resource_repository_alias} ${resource_repository_deps[resource_handle_type].field_name};
 % endfor
+% for resource_handle_type in resource_handle_types:
+  ::grpc::Status ConvertApiErrorStatusFor${resource_handle_type}(google::protobuf::int32 status, ${resource_handle_type} ${config["session_handle_parameter_name"]});
```

Comment:
Nit: Personally, the method name looks a little weird for RFmx to me: `ConvertApiErrorStatusForniRFmxInstrHandle`. I'm fine with it as is but consider capitalizing the `resource_handle_type` with something like `resource_handle_type[0].upper() + resource_handle_type[1:]`.

Reply:
Good suggestion